src/wp-includes/class-wp-theme-json.php

```diff
@@ -1078,7 +1078,7 @@ public function get_stylesheet( $types = array( 'variables', 'styles', 'presets'
 				$setting_nodes[ $root_settings_key ]['selector'] = $options['root_selector'];
 			}
 			if ( false !== $root_style_key ) {
-				$setting_nodes[ $root_style_key ]['selector'] = $options['root_selector'];
+				$style_nodes[ $root_style_key ]['selector'] = $options['root_selector'];
```

Comment:
Does this bug exist in the previous version that we addressed in this fix?

Reply:
I'm sorry, I'm not following that question, could you please rephrase it for me?

If you mean did this bug exist in prior versions of WP, then yes. It looks like the bug was first introduced into 6.2 [here](https://github.com/WordPress/wordpress-develop/pull/4013/files#diff-e12c3008d747d1bec5be9927c5e7b1ced0a88641abe52c278d495da936714817R989).